.gitignore

```diff
@@ -6,7 +6,6 @@ coverage.out
 *.iml
 *.swp
 *.log
-*.test.bin
```

Comment:
This should still be retained to avoid unnecessary submissions.

Reply:
Could you give me some example about 'submissions'?